src/cli/MeasureUpdateCommand.cpp

```diff
@@ -88,6 +97,202 @@ namespace cli {
+          ->group(helpOptionsGroupName);
+
+        // newMeasureSubCommand->footer(std::string{newMeasureExamples});
+      }
+
+      newMeasureSubCommand->callback([opt] {
+        // opt->newMeasureOpts.debug_print();
+
+        if (opt->newMeasureOpts.name.empty()) {
+          opt->newMeasureOpts.name = opt->newMeasureOpts.className;
+        }
+
+        BCLMeasure b(opt->newMeasureOpts.name, opt->newMeasureOpts.className, opt->newMeasureOpts.directoryPath, opt->newMeasureOpts.taxonomyTag,
+                     opt->newMeasureOpts.measureType, opt->newMeasureOpts.description, opt->newMeasureOpts.modelerDescription,
+                     opt->newMeasureOpts.measureLanguage);
+
+        fmt::print("Created a {} {} with class name '{}' in '{}'\n", opt->newMeasureOpts.measureLanguage.valueName(),
+                   opt->newMeasureOpts.measureType.valueName(), opt->newMeasureOpts.className,
+                   openstudio::toString(openstudio::filesystem::canonical(b.directory())));
+
+        std::exit(0);  // NOLINT(concurrency-mt-unsafe)
+      });
+    }
+
+    {
+
+      auto* copyMeasureSubCommand = measureCommand->add_subcommand("copy", "Copy a measure");
+      [[maybe_unused]] auto* existingDirectoryPathOpt =
+        copyMeasureSubCommand->add_option("EXISTING_DIRECTORY", opt->directoryPath, "Existing Directory for the measure")
+          ->option_text(" ")
+          ->required(true)
+          ->check(CLI::ExistingDirectory);
+      [[maybe_unused]] auto* newDirectoryPathOpt =
+        copyMeasureSubCommand->add_option("NEW_DIRECTORY", opt->newMeasureOpts.directoryPath, "New Directory for the measure")
+          ->option_text(" ")
+          ->required(true)
+          ->check(CLI::NonexistentPath);
+
+      copyMeasureSubCommand->callback([opt] {
+        boost::optional<BCLMeasure> b_;
+        try {
+          b_ = BCLMeasure(opt->directoryPath);
+        } catch (...) {
+          fmt::print(stderr, "Could not find a valid measure at '{}'\n", openstudio::toString(opt->directoryPath));
+          std::exit(1);
+        }
+        auto bClone_ = b_->clone(opt->newMeasureOpts.directoryPath);
+        if (bClone_) {
+          // Force updating the UID
+          bClone_->changeUID();
+          bClone_->checkForUpdatesFiles();
+          bClone_->checkForUpdatesXML();
+          bClone_->save();
+          fmt::print("Cloned the {} {} with class name '{}' from '{}' to '{}'\n", b_->measureLanguage().valueName(), b_->measureType().valueName(),
+                     b_->className(), openstudio::toString(openstudio::filesystem::canonical(b_->directory())),
+                     openstudio::toString(openstudio::filesystem::canonical(bClone_->directory())));
+          std::exit(0);  // NOLINT(concurrency-mt-unsafe)
+        } else {
+          fmt::print(stderr, "Something went wrong when cloning the measure");
+          std::exit(1);  // NOLINT(concurrency-mt-unsafe)
+        }
```

Comment:
Copying a measure. Exit gracefully if the directory isn't a valid measure.

Make sure we do end up with a new UID and everything is up to date in the measure.xml.